developer_guides/debugability/logger/index.rst

```diff
@@ -128,6 +128,15 @@ Examples
      sof-logger -l ldc_file -d
 
 
+.. note::
+  debugfs files used by ``sof-logger``
+
+  - ``etrace``: direct access (memcpy) to TRACE window of the SOF firmware
```

Comment:
```suggestion
  - ``etrace``: direct access to TRACE window of the SOF firmware in the shared memory "mailbox"
```

Reply:
@marc-hb, the trace window is not within the 'mailbox' memory. We have a bigger shared area and within that area we have the 'mailbox' (up and down region), trace, debug, stream, regs and exception windows.
Some of these windows are optional and only used for debugging, like the trace region.

Would it be better if we use the term 'region' instead of 'window' when referring to the TRACE window/region of the shared area?

Reply:
However the sof-logger's help documents the t option:
`sof-logger:      -t                     DMA 'trace' stream instead of 'etrace' error mailbox`

Probably something like this:
etrace: direct access to TRACE error mailbox of the firmware

Reply:
>  We have a bigger shared area and within that area we have the 'mailbox' (up and down region), trace, debug, stream, regs and exception windows.

Thanks, can you please share some pointers to this information? It looks like our terminology is inconsistent.

> However the sof-logger's help documents the t option:

_I_ recently changed this help message so it does not count :-) However I changed it based on symbols like `CONFIG_TRACEM` and `MAILBOX_TRACE_BASE` and other comments `git grep -i -e mailbox -e mbox -- src/trace/`; most of these are not from me.

So maybe "mailbox" is not the correct term here. On the other hand, "shared memory" is IMHO still better than "(memcpy)" in documentation.

Reply:
I would go with a simple:
`etrace: direct access to TRACE window of the SOF firmware`
If ever we will support non mem mapped type of DSP with SOF then this will be still valid.
We can use `region` instead of `window`, that should be fine as well.

The regions/windows are defined under xsram_window and they are really just sections within a shared memory between the CPU and DSP in currently supported platforms.
Imho to say that something is a mailbox it would take a bit more than a shared memory, a signaling mechanism like doorbells should be there. TRACE does not have associated doorbell afaik.

Reply:
@lyakh Have your change requests been completed? @ujfalusi Are you ready for the document to be merged? thanks

Reply:
> I would go with a simple: etrace: direct access to TRACE window of the SOF firmware

I feel like this is a confusing way to say "non-DMA trace" or "shared memory". What's wrong with either of these?

Reply:
> @lyakh Have your change requests been completed? @ujfalusi Are you ready for the document to be merged? thanks

@deb-intel yes, they have! I've approved the PR

Reply:
@marc-hb, can you check the new wording for etrace:
``etrace``: direct access to the shared TRACE window of the SOF firmware